### PR TITLE
"blip" login + header & footer changes (rebranding)

### PR DESCRIPTION
**Pull request by jebeck**
I've tentatively scheduled a design review with SK on Wednesday, but I believe this is ready for someone to start looking at in terms of code. It's mostly styling changes with the most substantial component changes being the addition of the `FooterLinks` component separate from the version footer (since there was a request to hide the links in some places but keep the version always present for debugging/support purposes).

Looking for feedback on both how and how much to try to DRY out some of the styles (see TODO comments). Note that since the bulk of this work was done before our meeting re: the style guide project, I have not introduced new styles (e.g., using `flex` to position footer elements) via CSS modules, but I suppose getting a start on that could be one method of DRYing out what's here. Somewhat a timing question. Do we want to wait for the larger new sign-up flow tasks (and larger styles updates per style guide) to start on that?

Will update from master as soon as current v0.17.0 in test is approved for prod.

### CODE REVIEW

**Comment by jebeck** [comment]
[<ux:1 dev:5 tst:1> [blip] Update Blip header and footer](https://trello.com/c/p5GATqC4/112-ux-1-dev-5-tst-1-blip-update-blip-header-and-footer)

File: app/components/footerlinks/footerlinks.less [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+.footer-link {
+  display: inline-block;
+  font-size: 14px;
```
Comment (hntrdglss) [inline app/components/footerlinks/footerlinks.less]: `@font-size-small`?

File: app/components/footerlinks/footerlinks.less [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  img {
+    margin: 0 12px;
+  }
+}
+
+@imgHeight: 32px;
+
+.footer-twitter {
+  width: 44.8px;
```
Comment (hntrdglss) [inline app/components/footerlinks/footerlinks.less]: 8/10ths of a pixel?! That's very specific. :)

File: app/components/version/version.less [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+.Version {
+  font-size: 14px;
```
Comment (hntrdglss) [inline app/components/version/version.less]: `@font-size-small`?

File: app/pages/app/app.less [hunk 4]
```diff
@@ -31,57 +31,27 @@
...
-      }
+    .footer-section {
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
 
-      &:first-child {
-        padding-left: 60px;
+      &.footer-section-top {
+        margin-bottom: 10px;
```
Comment (hntrdglss) [inline app/pages/app/app.less]: `@spacing-small`?

File: app/pages/emailverification/emailverification.less [hunk 5]
```diff
@@ -26,6 +47,7 @@
 }
 
 .EmailVerification-title {
+  margin-top: @spacing-large;
   margin-bottom: 20px;
```
Comment (hntrdglss) [inline app/pages/emailverification/emailverification.less]: `@spacing-base`?

File: app/pages/verificationwithpassword/verificationwithpassword.less [hunk 6]
```diff
@@ -13,15 +13,32 @@
...
+    justify-content: space-between;
+    .login-logo {
+      flex-grow: 2;
+    }
+  }
+}
+
 .VerificationWithPassword-form {
-  margin-bottom: @spacing-large;
+  margin-bottom: 5px;
```
Comment (hntrdglss) [inline app/pages/verificationwithpassword/verificationwithpassword.less]: `@spacing-tiny`?

File: app/pages/verificationwithpassword/verificationwithpassword.less [hunk 7]
```diff
@@ -13,15 +13,32 @@
...
   .input-group:nth-child(1) {
     color: @purple-blip;
     font-size: 28px;
     padding-bottom: 20px;
     margin-left: -10px;
     .input-group-explanation {
-      margin: 0px;
+      margin: 40px 0 0 0;
```
Comment (hntrdglss) [inline app/pages/verificationwithpassword/verificationwithpassword.less]: `@spacing-large 0 0 0`?

File: app/components/footerlinks/footerlinks.less (hunk 2 above)
Comment (jebeck) [inline app/components/footerlinks/footerlinks.less]: Math 🙂  I was scaling exactly to match the size of the Facebook logo from the original dimensions (slightly larger) of the Twitter logo.

**Comment by jebeck** [comment]
Ping @hntrdglss for final review - I believe I've addressed all feedback. Thanks for the catches on swapping in variables, and I factored out a mixin for the flexbox layout on the logged-out routes.

**Comment by jebeck** [comment]
Closing with #396 replacing due to tag issue...

**Pull request closed without merging**